models/project/issue.go

```diff
@@ -33,6 +34,50 @@ func deleteProjectIssuesByProjectID(ctx context.Context, projectID int64) error
 	return err
 }
 
+type ErrProjectIssueNotExist struct {
+	IssueID int64
+}
+
+func (e ErrProjectIssueNotExist) Error() string {
+	return fmt.Sprintf("can't find project issue [issue_id: %d]", e.IssueID)
+}
+
+func IsErrProjectIssueNotExist(e error) bool {
+	_, ok := e.(ErrProjectIssueNotExist)
+	return ok
+}
+
+func GetProjectIssueByIssueID(ctx context.Context, issueID int64) (*ProjectIssue, error) {
+	issue := &ProjectIssue{}
+
+	has, err := db.GetEngine(ctx).Where("issue_id = ?", issueID).Get(issue)
+	if err != nil {
+		return nil, err
+	}
+
+	if !has {
+		return nil, ErrProjectIssueNotExist{IssueID: issueID}
+	}
+
+	return issue, nil
+}
+
+func (issue *ProjectIssue) LoadProjectBoard(ctx context.Context) error {
+	if issue.ProjectBoard != nil {
+		return nil
+	}
+
+	var err error
+
+	if issue.ProjectBoardID == 0 {
```

Comment:
Since now the default column will be stored in database, so I don't think we should assume 0 means default column.

Reply:
Are you sure? I seen this in the code comment, I tink it's better to keep this status to prevent bug.